homeassistant/components/onkyo/media_player.py

```diff
@@ -236,8 +314,18 @@ def __init__(
         self._source_list = list(sources.values())
         self._source_mapping = sources
         self._reverse_mapping = {value: key for key, value in sources.items()}
+        self._sound_mode = None
+        self._sound_mode_list = list(SOUND_MODE_MAPPING.keys())
```

Comment:
Keys is the default iterator.
```suggestion
        self._sound_mode_list = list(SOUND_MODE_MAPPING)
```

Reply:
Agreed, and thank you for the review!